packages/cli/generation/local-generation/local-workspace-runner/src/runLocalGenerationForWorkspace.ts

```diff
@@ -119,7 +125,22 @@ async function writeFilesToDiskAndRunGenerator({
     if (firstLocalOutputItem == null) {
         return;
     }
-    await rm(absolutePathToLocalOutput, { force: true, recursive: true });
+    // await rm(absolutePathToLocalOutput, { force: true, recursive: true });
+    const dotFernIgnoreFile = path.join(absolutePathToLocalOutput, ".fernignore");
```

Comment:
nit: rename variable to `absolutePathToFernignore`

Reply:
Awesome. I will